test/pylib/manager_client.py

```diff
@@ -279,7 +283,7 @@ async def servers_add(self, servers_num: int = 1,
         assert servers_num > 0, f"servers_add: cannot add {servers_num} servers, servers_num must be positive"
 
         try:
-            data = self._create_server_add_data(None, cmdline, config, property_file, start, expected_error)
+            data = self._create_server_add_data(None, cmdline, config, property_file, start, None, expected_error)
```

Comment:
Additionally adding `seeds` to `servers_add` is trivial and might be useful later (especially for someone who doesn't know test.py internals and won't extend the API easily). This PR doesn't need this change though, it's optional.

Reply:
Yes, can do that if we need to change something else, would like to avoid rewriting git/rebuilding/CI run only for that.